lnbits/settings.py

```diff
@@ -360,6 +360,7 @@ class SaaSSettings(LNbitsSettings):
 
 
 class PersistenceSettings(LNbitsSettings):
+    first_install: bool = Field(default=False)
```

Comment:
This is not a `PersistenceSettings` (related to how data is persisted).
Maybe `TransientSettings` is more appropriate.:

```py
class TransientSettings(InstalledExtensionsSettings):
    # Transient Settings:
    #  - are initialized, updated and used at runtime
    #  - are not read from a file or from the `settings` table
    #  - are not persisted in the `settings` table when the settings are updated
    #  - are cleared on server restart
```

Reply:
the comment sounds fitting but why is it inherited from `InstalledExtensionsSettings`?

Reply:
How should I treat this?

Reply:
> How should I treat this?

Just move it under `TransientSettings`